notebooks/255-bark-text-to-audio/255-bark-text-to-audio.ipynb

```diff
@@ -0,0 +1,1143 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip install "torch=1.13.1" "torchvision" "torchaudio" "openvino&gt;=2023.1.0" gradio
</pre><p>-q and torch==1.13.1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>